python/lsst/ts/IntegrationTests/at_state_transition_configs.py

```diff
@@ -0,0 +1,133 @@
+# This file is part of ts_IntegrationTests.
```

Comment:
I'd call this auxtel_state... for clarity.

Reply:
Fixed